lib/router/index.js

```diff
@@ -53,6 +54,18 @@ routerEvents.forEach((event) => {
   })
 })
 
+const warnAboutRouterOnAppUpdated = execOnce(() => {
+  console.warn(`Router.onAppUpdated is removed - visit https://err.sh/next.js/no-on-app-updated-hook for more information.`)
```

Comment:
`https://err.sh/next.js/no-on-app-updated-hook` (otherwise consoles might not linkify it)

Reply:
This is done.